judge/migrations/0114_contest_scoreboard_visibility.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.19 on 2021-04-05 03:37
+
+from django.db import migrations, models
+
+
+def hide_scoreboard_eq_true(apps, schema_editor):
+    Contest = apps.get_model('judge', 'Contest')
+    Contest.objects.filter(hide_scoreboard=True).update(scoreboard_visibility='C')
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0113_contest_decimal_points'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='contest',
+            name='scoreboard_visibility',
+            field=models.CharField(choices=[('V', 'Visible'), ('C', 'Hidden for duration of contest'), ('P', 'Hidden for duration of participation')], default='V', help_text='Scoreboard visibility through the duration of the contest', max_length=64, verbose_name='scoreboard visibility'),
+        ),
+        migrations.RunPython(hide_scoreboard_eq_true, atomic=True),
```

Comment:
You're missing the [reverse operation](https://docs.djangoproject.com/en/2.2/ref/migration-operations/#django.db.migrations.operations.RunPython).